python/vowpalwabbit/pyvw.py

```diff
@@ -501,6 +501,25 @@ def get_weight(self, index, offset=0):
         """
         return pylibvw.vw.get_weight(self, index, offset)
 
+    def get_weight_from_name(self, feature_name, namespace_name=""):
```

Comment:
Maybe it makes sense for the default `namespace_name` to be `" "` since in text format it would correspond to `| feature`. " " is the default namespace

Reply:
Yep sure !

Oddly it works whether the namespace name is a empty string `""`, one space `" "` or multiple spaces.